Lib/test/support/__init__.py

```diff
@@ -155,8 +155,12 @@ def get_attribute(obj, name):
     try:
         attribute = getattr(obj, name)
     except AttributeError:
-        raise unittest.SkipTest("module %s has no attribute %s" % (
-            obj.__name__, name))
+        if isinstance(obj, type(sys)):
```

Comment:
I guess that the code is the same in master. I am surprised that types.ModuleType is not used here.